lib/sidechain.js

```diff
@@ -201,10 +201,14 @@ function deserializeFtOutputs(buf, offset) {
     offset += 32;
     var scid = buf.slice(offset, offset + 32).reverse().toString('hex');
     offset += 32;
+    var mcReturnAddress = buf.slice(offset, offset + 20).reverse().toString('hex');
```

Comment:
this in reality is the pubkeyhash, should be nice to derive the address from it because the zendoo output displays the address